src/constants.ts

```diff
@@ -4,3 +4,5 @@ export const ALFAJORES_CHAIN_ID = 44787
 export const WEBHOOK_RECIPIENT_BASE_URL =
   'https://liquidity-dot-celo-mobile-alfajores.appspot.com'
 export const WEBHOOK_RECIPIENT_HISTORY_ENDPOINT = '/fiatconnect/webhook/history'
+export const BASE64_IMAGE =
```

Comment:
this is not a valid base64 image

Reply:
it has symbols not included in the [base64 alphabet](https://en.wikipedia.org/wiki/Base64) and failed to decode when I tried to put it into an online base64 image decoder

Reply:
if you just need some base64 image you could copy an illustration of an ID card, which would be smaller than a "real" picture and not contain any PII